zerver/tests/test_logging_handlers.py

```diff
@@ -10,8 +10,7 @@
 from django.utils.log import AdminEmailHandler
 from functools import wraps
 from mock import patch
-if False:
-    from mypy_extensions import NoReturn
+from mypy_extensions import NoReturn
```

Comment:
@neiljp merci for the points; I had just narrowed down the cause to `NoReturn` not being imported, now. This is strange considering that `mypy` should have screamed out the undefined error.

Travis should be all green now. It is green in my branch's Travis.

Reply:
The problem is mypy-specific, but it went undetected.